docs/usage/configuration-options.md

```diff
@@ -748,10 +748,6 @@ See [Private module support](https://docs.renovatebot.com/getting-started/privat
 
 ## excludeCommitPaths
 
-<!-- prettier-ignore -->
-!!! warning
-    For advanced users only!
```

Comment:
i think there are some more to change. why not doing all in one pr?

Reply:
which changes?

Reply:
https://github.com/renovatebot/renovate/blob/867c47ff6542fa36789b574f216f31aa13cdc0a8/docs/usage/configuration-options.md#L396

and below

Reply:
https://github.com/renovatebot/renovate/blob/867c47ff6542fa36789b574f216f31aa13cdc0a8/docs/usage/configuration-options.md#L864-L869

https://github.com/renovatebot/renovate/blob/867c47ff6542fa36789b574f216f31aa13cdc0a8/docs/usage/configuration-options.md#L935-L940

https://github.com/renovatebot/renovate/blob/867c47ff6542fa36789b574f216f31aa13cdc0a8/docs/usage/configuration-options.md#L1197-L1201